test/interop-test.js

```diff
@@ -2,34 +2,34 @@
 'use strict'
 
 const { expect } = require('aegir/utils/chai')
-const mh = require('multihashing-async').multihash
-const CID = require('cids')
+const { CID } = require('multiformats')
 const Key = require('interface-datastore').Key
 const uint8ArrayToString = require('uint8arrays/to-string')
+const uint8ArrayFromString = require('uint8arrays/from-string')
 
 /**
  * @param {import('../src/index')} repo
  */
 module.exports = (repo) => {
   describe('interop', () => {
     it('reads welcome-to-ipfs', async () => {
-      const welcomeHash = mh.fromHexString(
-        '1220120f6af601d46e10b2d2e11ed71c55d25f3042c22501e41d1246e7a1e9d3d8ec'
+      const welcomeHash = CID.decode(
+        uint8ArrayFromString('1220120f6af601d46e10b2d2e11ed71c55d25f3042c22501e41d1246e7a1e9d3d8ec', 'base16')
       )
 
-      const val = await repo.blocks.get(new CID(welcomeHash))
-      expect(uint8ArrayToString(val.data)).to.match(/Hello and Welcome to IPFS/)
+      const val = await repo.blocks.get(welcomeHash)
+      expect(uint8ArrayToString(val)).to.match(/Hello and Welcome to IPFS/)
```

Comment:
`bytes` also has a `fromString()` and `toString()` that does utf-8 with `TextEncoder` and `TextDecoder` similar to this btw